app/src/main/java/mollie/tictactoe/StateConverter.java

```diff
@@ -0,0 +1,66 @@
+package mollie.tictactoe;
+
+import mollie.tictactoe.players.MobilePlayer;
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.GameEngine;
+import ttt.game.Marks;
+
+public class StateConverter {
+
+    private static final String CURRENT_MARK = "currentMark";
+    private static final String BOARD = "board";
+
+    public static String[] convertBoardToString(Board board) {
```

Comment:
`List<String>` has a nicer, more modern API than `String []`. It allows you to do fancy iteration, mapping, and searching. Hurray for the List!